signingscript/src/signingscript/sign.py

```diff
@@ -834,6 +835,14 @@ def write_signing_req_to_disk(fp, signing_req):
     fp.write(b"}]")
 
 
+def write_files_signing_req_to_disk(fp, signing_req):
+    """Write files/ signing_req to fp.
+    The files content should be ecoded as base64 strings
+    """
+    encoded_signing_req = json.dumps(signing_req).encode("utf-8")
+    fp.write(encoded_signing_req)
```

Comment:
Won't we potentially hit the memory issues that `write_signing_req_to_disk` was trying to solve now? Maybe there's a good reason to be less concerned in this case?

Reply:
True. I replaced this with a chunking version